src/opera/commands/unpackage.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import uuid
+
+from pathlib import Path
+
+from opera.error import DataError, ParseError
+from opera.parser.tosca.csar import CloudServiceArchive
+
+
+def add_parser(subparsers):
+    parser = subparsers.add_parser(
+        "unpackage",
+        help="Unackage TOSCA CSAR to a specified location"
+    )
+    parser.add_argument(
+        "--destination", "-d",
+        help="Path to the location where the CSAR file will be extracted to",
+    )
+    parser.add_argument(
+        "--format", "-f", choices=("zip", "tar"),
+        default="zip", help="CSAR compressed file format",
+    )
+    parser.add_argument("csar", help="Path to the TOSCA Cloud service archive")
+    parser.set_defaults(func=_parser_callback)
+
+
+def _parser_callback(args):
+    if not Path(args.csar).is_file():
```

Comment:
I kept the string because `shutil.unpack_archive` accepts an archive path and destination folder path as strings. But if you think that it's better to have a pathlib object I can modify that (and call p.absolute() when passing it to the `unpack_archive` function).

Reply:
I am using path objects now, take a look and specify if you want me to also pass them to the util/csar functions that invoke the actions.

Reply:
This can be also resolved. We found out that [`shutil.unpack_archive`](https://docs.python.org/3/library/shutil.html#shutil.unpack_archive)  function can't receive Path objects for python 3.6 (:disappointed:) so we can keep things as they are.